Assets/HoloToolkit/Input/Scripts/Utilities/ControllerInfo.cs

```diff
@@ -11,10 +11,9 @@ namespace HoloToolkit.Unity.InputModule
     /// </summary>
     public class ControllerInfo : MonoBehaviour
     {
-        // TODO: Implement all controller components.
-        //private GameObject home;
-        //private Transform homePressed;
-        //private Transform homeUnpressed;
+        private GameObject home;
```

Comment:
Are we actually using these references now?

Reply:
I added an AnimateHome method to use them. I never call it, since the home button will leave the app when pressed, but it's useful to keep the references to the home button info while removing the "unused" warnings.